docs/ai_actions/work_with_ai_actions.md

```diff
@@ -0,0 +1,100 @@
+
+## View AI actions
+
+With administrator permissions, you can view all AI actions configured in the application by navigating to the Admin Panel and selecting **AI actions**.
+
+![AI actions in Admin Panel](img/ai_actions_list.png)
+
+You can narrow down the list of AI actions by filtering it by the status, either Enabled or Disabled, or by the type.
+Out of the box, there are two types of AI Actions present in the system:
+
+- **Refine text** - used primarily in [online editor](create_edit_content_items.md#ai-assistant) for performing text-to-text actions, for example: "Rewrite text in formal tone"
+- **Generate alternative text** - used in the [image asset editing screen](upload_images.md#ai) to perform image-to-text actions, for example: "Generate short alternative description of an image"
+
+It may happen that a set of sample AI actions has been [installed with the AI Actions package]([[= developer_doc =]]/ai_actions/install_ai_actions/#install-sample-ai-action-configurations-optional), and there is a number of existing AI actions that you can modify and clone.
+
+!!! note "Custom action types"
+
+    In your specific case, the types available can be different, and your organization's development team can create custom AI action types.
+    For more information, see [developer documentation]([[= developer_doc =]]/ai_actions/).
+
+### View AI action details
+
+Navigate to the Admin Panel and select **AI actions**.
+In the **AI actions** list, click the name of an AI action to review its details.
+For example, in the **Properties** tab, you can see specific settings that modify the prompt that is sent to an AI service.
+
+![AI action details](img/ai_action_details.png)
+
+## Edit existing AI actions
+
+You can modify the existing AI actions.
+
+1\. Navigate to the Admin Panel and select **AI actions**.
+
+2\. In the **AI actions** list, click the **Edit** icon next to a name of the AI action that you want to modify.
+
+3\. In the **Global properties** section, you can change the name and description of the AI action. You can also toggle the availability of the AI action between disabled and enabled.
+
+4\. In the **Settings** area, change the settings that modify the behavior of an AI service that executes an AI action, for example:
+
+- **Prompt** - modifies the default request by passing a verbal command, for example, "Make it short and formal."
+
+!!! note "Default request"
+
+    The default request can be seen at the top of the settings area, on a light blue background.
+
+- **Max tokens** - sets a maximum number of "[words](https://help.openai.com/en/articles/4936856-what-are-tokens-and-how-to-count-them)" or tokens that can be used in a single request by both the request and the response
+
+- **Length of prompt output** -  sets a maximum number of words or tokens that can be used by the response
```

Comment:
I'd simplify this: it's not really about tokens, but about the maximum length of the generated result.

Reply:
OK, I relied on a definition that I've found in Open AI's specification